### PR TITLE
feat: Ungrid filter row to help with lemonizing

### PR DESCRIPTION
**Pull request by benjackwhite**
## Problem

Wanted to try and help with some property filter styling as its been frustrating me as well for quite a while

## Changes

* It's not quite as fixed as the previous grid system was but it is way more **flex**-able which I think overall wins from a UX perspective

### Large screen
Screenshot 2022-09-09 at 13 18 57

### Small screen
Screenshot 2022-09-09 at 13 19 13

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

**Comment by liyiy** [comment]
Thanks for taking a look into this! I think this definitely helps but also causes some other issues?

Screen Shot 2022-09-08 at 1 58 07 PM

**Comment by benjackwhite** [comment]
@liyiy I updated some more styles. Not sure what exactly you were referring to but the style of the squished multiselect is the same as on prod. That component may take a little more effort to refactor but should probably simplified around a `LemonMultiSelect` at some point

File: frontend/src/styles/global.scss [hunk 1]
```diff
@@ -553,6 +553,7 @@ input::-ms-clear {
     min-height: 40px;
     border: 1px solid var(--border);
     border-radius: var(--radius);
+    background-color: var(--white);
```
Comment (mariusandra) [inline frontend/src/styles/global.scss:556]: Somehow this feels wrong. E.g. what if we want dark mode? Could `var(--bg-light)` work better?

**Review by mariusandra — APPROVED** [review]
Always good to make more ant lemonade. Found one question about color (read: colour) that might be worth looking into.

File: frontend/src/styles/global.scss (hunk 1 above)
Comment (benjackwhite) [reply frontend/src/styles/global.scss]: Yeah --bg-light should be used 👍 

I think this won't help us that much regarding dark mode though 😆

**Review by liyiy — APPROVED** [review]
Looks so good! Thanks for helping out with this 🤗

**Pull request merged**